test/run_envoy_tests.sh

```diff
@@ -20,15 +20,33 @@ fi
 : ${TEST_TMPDIR:=$TEST_BASE/tmp}
 : ${TEST_SRCDIR:=$TEST_BASE/runfiles}
 export TEST_TMPDIR TEST_SRCDIR
+export TEST_WORKSPACE=""
+export TEST_UDSDIR="$TEST_TMPDIR"
 
 echo "TEST_TMPDIR=$TEST_TMPDIR"
 echo "TEST_SRCDIR=$TEST_SRCDIR"
 
 mkdir -p $TEST_TMPDIR
 
 $SOURCE_DIR/test/certs/gen_test_certs.sh $TEST_SRCDIR/test/certs
-$SOURCE_DIR/test/config/integration/gen_test_configs.sh $SOURCE_DIR/test/config/integration \
-  $TEST_SRCDIR/test/config/integration
+
+# Some hacks for the config file template substitution. These go away in the Bazel build.
+CONFIG_IN_DIR="$SOURCE_DIR"/test/config/integration
+CONFIG_RUNFILES_DIR="$TEST_SRCDIR/$TEST_WORKSPACE"/test/config/integration
+CONFIG_OUT_DIR="$TEST_TMPDIR"/test/config/integration
+mkdir -p "$CONFIG_RUNFILES_DIR"
+mkdir -p "$CONFIG_OUT_DIR"
+cp "$CONFIG_IN_DIR"/*.json "$CONFIG_RUNFILES_DIR"
```

Comment:
What is the point of copying any of this into runfiles first? What is the difference between runfiles and tmp? Sorry this is kind of hard to follow.